modules/persistent-storage-csi-vsphere-disable-storage-procedure.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-disable-storage-procedure_{context}"]
+= Disabling and enabling storage on vSphere
+
+.Procedure
+
+To disable or enable storage on vSphere:
+
+. Click *Administration* > *CustomResourceDefinitions*.
+
+. On the *CustomResourceDefinitions* page next to the *Name* dropdown box, type "clustercsidriver".
+
+. Click *CRD ClusterCSIDriver*.
+
+. Click the *Instances* tab.
+
+. Click *csi.vsphere.vmware.com*.
+
+. Click the *YAML* tab.
+
+. For `spec.managementState`, change the value to `Removed` or `Managed`:
++
+* `Removed`: storage is disabled
+* `Managed`: storage is enabled
+
+. Click *Save*.
```

Comment:
Can we add a step that shows how to confirm that the driver is indeed removed @mpatlasov 
`oc get pod -n openshift-cluster-csi-drivers `
should only return the operator

Reply:
Done.

Reply:
Done.